src/audio-context-mock.ts

```diff
@@ -44,12 +44,15 @@ export class AudioContextMock extends EventTarget implements IAudioContext {
 
     private _options: IAudioContextOptions;
 
+    private _state: TAudioContextState;
+
     constructor(options: IAudioContextOptions = {}) {
         super();
 
         this._deLorean = new DeLorean();
         this._onstatechange = null;
         this._options = options;
+        this._state = <TAudioContextState>'running';
```

Comment:
The spec actually requires the `AudioContext` to be `suspended` at first. Could you change the initial state to be suspended?

```ts
this._state = <TAudioContextState>'suspended';
```